tools/src/main/java/org/apache/kafka/tools/ConsumerPerformance.java

```diff
@@ -310,7 +318,7 @@ public ConsumerPerfOptions(String[] args) {
                 .describedAs("milliseconds")
                 .ofType(Long.class)
                 .defaultsTo(10_000L);
-            numMessagesOpt = parser.accepts("messages", "REQUIRED: The number of messages to consume.")
```

Comment:
Wouldn't it be better to introduce a deprecation cycle for this option, like `consumer.config`?